docs/03-extensions/payzen/index.mdx

```diff
@@ -0,0 +1,54 @@
+---
+title: "Payzen"
+description:
+  "The Payzen extension allows customer to pay using Payzen."
+---
+
+<p>{frontMatter.description}</p>
+
+:::danger
+
+This extension is only compatible with Magento1/OpenMage, we are working on bring compatibility to Magento2.
+
+:::
+
+## Installation
+
+First, you need to install the `@front-commerce/payzen` package:
+
+```bash title="Command to install @front-commerce/payzen"
+$ pnpm install @front-commerce/payzen
```

Comment:
```suggestion
$ pnpm add @front-commerce/payzen
```

It seems that most of our install commands are incorrect. Can you followup with a fix on `main` for the other ones please?